src/app/utils.ts

```diff
@@ -4,10 +4,10 @@ export const GRAPH_REFRESH_INTERVAL = 5000; // milliseconds
 export const STATS_HISTORY_TIME_FILTER = '10'; // minutes
 export const ASSET_READINGS_TIME_FILTER = 600; // seconds
 export const COLOR_CODES = ['#3498DB', '#85C1E9', '#239B56', '#82E0AA', '#B03A2E', '#F1948A', '#FF8C00', '#FF0000',
```

Comment:
I believe, We can remove this now. However for R/G/B (e-pHat) datapoints/ attributes, let's keep matching colors as we do in current develop for legends / graph lines